src/elements/content-sidebar/activity-feed/task-new/__tests__/AssigneeList-test.js

```diff
@@ -0,0 +1,187 @@
+                        type: 'user',
+                        id: '111',
+                        name: 'AL',
+                    },
+                    status: TASK_NEW_REJECTED,
+                }),
+                next_marker: 'abc',
+                limit: pageSize,
+            };
+            const wrapper = shallow(
+                <AssigneeList
+                    onExpand={onExpand}
+                    users={paginatedAssignees}
+                    minAssignees={max}
+                    getAvatarUrl={mockGetAvatarUrl}
+                />,
+            );
+            const expandBtn = global.queryAllByTestId(wrapper.dive(), 'show-more-assignees');
+            const hideBtn = global.queryAllByTestId(wrapper.dive(), 'show-less-assignees');
+
+            expect(expandBtn).toHaveLength(1);
+            expect(hideBtn).toHaveLength(0);
+            expect(expandBtn.find('FormattedMessage').prop('values')).toEqual({ additionalAssigneeCount: '17+' });
+        });
+
+        test('should show not show overflow icon when there are fewer assignees than minAssignees', () => {
+            const max = 3;
+            const wrapper = shallow(
+                <AssigneeList
+                    onExpand={onExpand}
+                    users={assignees}
+                    minAssignees={max}
+                    getAvatarUrl={mockGetAvatarUrl}
+                />,
+            );
+            expect(global.queryAllByTestId(wrapper.dive(), 'show-more-assignees')).toHaveLength(0);
+            expect(global.queryAllByTestId(wrapper.dive(), 'show-less-assignees')).toHaveLength(0);
+        });
+
+        test('should open assignee list when expand button is clicked', () => {
+            const max = 2;
+            const wrapper = mount(
+                <AssigneeList
+                    onExpand={onExpand}
+                    users={assignees}
+                    minAssignees={max}
+                    getAvatarUrl={mockGetAvatarUrl}
+                />,
+            );
+            const expandBtn = global.queryAllByTestId(wrapper, 'show-more-assignees').first();
+            expandBtn.simulate('click');
+
+            const assigneeList = global.queryAllByTestId(wrapper, 'task-assignee-list');
+            expect(assigneeList.children()).toHaveLength(3);
+        });
+
+        test('should hide assignee list when hide button is clicked', () => {
+            const max = 2;
+            const wrapper = mount(
+                <AssigneeList
+                    onExpand={onExpand}
+                    users={assignees}
+                    minAssignees={max}
+                    getAvatarUrl={mockGetAvatarUrl}
+                />,
+            );
+
+            const expandBtn = global.queryAllByTestId(wrapper, 'show-more-assignees').first();
+            expandBtn.simulate('click');
+
+            let assigneeList = global.queryAllByTestId(wrapper, 'task-assignee-list');
+            expect(assigneeList.children()).toHaveLength(3);
```

Comment:
could you add the testid to the task-assignee items themselves so you don't have to check `.children()`, which could have some other elements in there?